spec/models/manageiq/providers/amazon/cloud_manager/vcr_specs/refresher_inventory_object_spec.rb

```diff
@@ -32,6 +33,7 @@
           end
 
           assert_common
+          assert_mapped_tags_on_vm
```

Comment:
could we move this inside of the specific Vms asserts?

Especially  assert_specific_vm_powered_on, assert_specific_vm_on_cloud_network_public_ip and assert_specific_vm_on_cloud_network

That way it will be automatically tested in old, new and targeted refresh

Reply:
done for vms

For templates the only label I found at all is `Name` :neutral_face: 
and both `assert_specific_template*` have 0 labels (not even `Name`) so only calling this function explicitly in some tests.
This means targeted image taggings refresh is not well tested.